packages/apply/index.ts

```diff
@@ -0,0 +1,396 @@
+import {
+  readPkgFile,
+  writePkgFile,
+  testSerializePkgFile,
+  PackageFile,
+  ConfigFile,
+} from "@covector/files";
+import { compareBumps, CommonBumps } from "@covector/assemble";
+import semver from "semver";
+import cloneDeep from "lodash.clonedeep";
```

Comment:
this is dangerous, because AFAIK this has not been updated since many, many vuln disclosures. you should better `import { cloneDeeep } from "lodash";` if it is still available.

Reply:
Oh wow, that one is really old. I was trying not to pull in the whole of lodash, but... It kind of seems like the lodash project has lost its energy. I am tempted to soft fork the couple of small functions that I am currently using.

Reply:
yeah, the thing is that there are tons of vulns if you don't keep up with bleeding edge...

Reply:
https://snyk.io/vuln/npm:lodash

Reply:
You have to use https://snyk.io/test/npm/lodash@4.17.21 - but lodash vulnerability as a service is almost sort of a running gag :(

Reply:
ugh, yea.... true. I'll punt in this PR, but we should do it for the next version.

Reply:
https://github.com/jbolda/covector/pull/155